projects/iio_demo/src/hal/stm32f4xx_hal_conf.h

```diff
@@ -0,0 +1,486 @@
+/**
```

Comment:
Is this a generated source, or source from somewhere else? I am thinking that maybe there is a way to avoid having it whole here if we don't need all of it.

Reply:
Everything in hal/ is generated, including this file, and this file is very important in activating/deactivating stuff in the STM32 HAL. The only change I applied on the hal/* files is astyle.